src/util/fs_utils.js

```diff
@@ -159,6 +159,14 @@ function create_path(dir, mode) {
     return fs.promises.mkdir(dir, { mode, recursive: true });
 }
 
+function create_file(file_path) {
+    return fs.promises.open(file_path, 'w');
+}
+
+function create_symbolic_link(target, link_path) {
```

Comment:
I think that the added utils here are redundant. it is a one-liner and only used in the test. 
if a function is necessary (maybe for `create_file` where you also need to close the file) you can make a local func in the test

Reply:
Done. Moved to the test.